securing_apps/topics/oidc/nodejs-adapter.adoc

```diff
@@ -42,13 +42,30 @@ ifeval::[{project_product}==true]
 
 endif::[]
 
+Optional we can use git in order to track our changes:
+----
+    git init
+----
+In that case create a file called `.gitignore` at the root folder of your project and add below line in order to exclude tracking node_modules directory
+----
+    node_modules/
+----
+
+Lets make our first commit:
+----
+git add .
+git commit -m 'Initial commit'
+----
+
```

Comment:
I'd suggest removing the entire block above. Cover `git` steps is out of the scope for the Keycloak, plus Node.js developers are already familiar with `.gitignore`.